### ISSUE TITLE
Is this plugin still maintained?

### ISSUE THREAD

**Issue by oupala** [bubkoo/hexo-toc#24]
The activity on this git repository seems to get flat this months.

Is this plugin still maintained?

**Comment by noraj** [comment]
Seems like more one year without a commit or answer on issues.

**Comment by oupala** [comment]
Yes, you're right. And that makes me doubt about hexo. Maybe I should use another static website generator, such as hugo.

Or there might be an alternative to hexo-toc?

**Comment by noraj** [comment]
@oupala Hexo is good even if some plugins are not maintained sometimes. But for TOC a plugin is not mandatory: see http://hsin919.github.io/2015/02/11/Add-TOC/, you can use an external tool such as doctoc or add this in the template layout of your theme the following piece of code.

```ejs
<% if(post.toc !== false){ %>
<div id="toc" class="toc-article">
	<strong class="toc-title">Table of contents</strong>
<%- toc(post.content) %>
</div>
<% } %>
```

**Comment by oupala** [comment]
I agree TOC could be generated on the writer side, when saving. But generating the TOC on static generation step, automatically, whatever the writers have installed or not, is a must have.

On the contrary, we might want to have a TOC on long articles only, which is not feasible when modifying the template.

Thanks anyway for pointing out workarounds.
